### PR TITLE
Fixing No Projects Found

### PR DESCRIPTION
**Pull request by EthanGu4**
### Description
[//]: # "[Required] Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- Added image and description for when no projects are found from the search query
- Created new variable hasProjects that detects if any projects are found given the search
- If hasProjects is false, we produce the layout provided in the sprint, as well as the styling that goes with it
- Added image NoProjectsFound in assets/SearchBar-Icons/icons

### Screenshots
[//]: # "Add any screenshots that will help your PR reviewer understand what’s going on in the PR!"
[//]: # "[Required] For frontend tasks: add screenshots of the layout and expected behavior (GIFs if you’re feeling extra fancy)!"
[//]: # "Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

## How to review
[//]: # "/[Required] Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"
- Look at the functionality of hasProjects
- Make sure all the styling looks the same as in the figma

## Next steps
[//]: # "[Optional] - What has not yet been added to the PR, doesn’t yet have any functionality — essentially, what needs to be fixed or added."
[//]: # "[Optional] any temporary fixes in this PR that should be cleaned up later."
- lowkey the modal on the website looks wider than the one on the figma but idk

### Online sources
[//]: # "[Optional] insert links to any useful tutorials or documentation that you used for this PR!"

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @jjstnlee

### CODE REVIEW

File: components/ProjectsListingModal/styles.ts [hunk 1]
```diff
@@ -55,10 +56,35 @@ export const ModalContents = styled.div`
...
+  justify-content: center;
+  text-align: center;
+  flex: 1;
+  padding-top: 10.563rem;
+  gap: 0.625rem;
+`;
+
+export const NoProjectsFoundText = styled.div`
+  ${CoinbaseText.style}
+  color: var(--navy-85, #4C5671);
```
Comment (jjstnlee) [inline components/ProjectsListingModal/styles.ts:77]: use `COLORS.navy85`

File: components/ProjectsListingModal/styles.ts [hunk 2]
```diff
@@ -55,10 +56,35 @@ export const ModalContents = styled.div`
...
+  font-size: 14px;
```
Comment (jjstnlee) [inline components/ProjectsListingModal/styles.ts:78]: change to rem

File: components/ProjectsListingModal/styles.ts [hunk 3]
```diff
@@ -55,10 +56,35 @@ export const ModalContents = styled.div`
...
+  font-style: normal;
+  font-weight: 300;
+  line-height: normal;
+
+  max-width: 273px;
```
Comment (jjstnlee) [inline components/ProjectsListingModal/styles.ts:83]: change to rem

File: components/ProjectsListingModal/styles.ts [hunk 4]
```diff
@@ -55,10 +56,35 @@ export const ModalContents = styled.div`
...
+  max-height: 49px;
```
Comment (jjstnlee) [inline components/ProjectsListingModal/styles.ts:84]: change to rem

**Review by jjstnlee — CHANGES_REQUESTED** [review]

**Review by jjstnlee — APPROVED** [review]
